shared/src/main/scala/com/cibo/evilplot/plot/components/Axes.scala

```diff
@@ -213,30 +217,132 @@ object Axes {
   trait AxesImplicits {
     protected val plot: Plot
 
+    /** Add a continuous axis to the plot.
+      * @param boundsFn         Takes a plot and returns the bounds this axis will display.
+      * @param position         The side of the plot to add the axis.
+      * @param tickCount        The number of tick lines.
+      * @param tickRenderer     Function to draw a tick line/label.
+      * @param labelFormatter   Custom function to format tick labels.
+      * @param tickCountRange   Allow searching over axis labels with this many ticks.
+      * @param updatePlotBounds Set plot bounds to match the axis bounds.
+      * @param fixedBounds      Force ticks to match bounds.
+      */
+    def continuousAxis(
+      boundsFn: Plot => Bounds,
+      position: Position,
+      tickCount: Option[Int] = None,
+      tickRenderer: Option[TickRenderer] = None,
+      labelFormatter: Option[Double => String] = None,
+      tickCountRange: Option[Seq[Int]] = None,
+      updatePlotBounds: Boolean = true,
```

Comment:
This stuff always confuses me, so I want to clarify: is `updatePlotBounds` setting the plot's transformation to match the axis bounds or the axis label bounds?

Reply:
It doesn't affect the transformation (except I suppose indirectly through setting the plot bounds). If `updatePlotBounds` is true, then the existing behavior is used which updates the plot bounds to match the axis bounds (AxisDescriptor#axisBounds).
```scala
// new implementation of this behavior
component +: plot.ybounds(component.getDescriptor(plot, fixedBounds).axisBounds)
// old replaced version
//component +: plot.ybounds(component.getDescriptor(plot, plot.yfixed).axisBounds)
```
If `updatePlotBounds` is false, then the axis component is added to the plot without touching the plot bounds at all.
```scala
component +: plot
```